roadmap/c-unwind-abi.md

```diff
@@ -16,30 +16,24 @@ compilers use.
 
 **Warning:** We are still in the process of fully specifying when and
 how "unwinding interop" works between native functions and Rust. This
-roadmap item alone **only** adds the ABI -- it does not define what
-should happen if unwinding actually occurs, and hence **even if you
-use this ABI**, unwinding across a `"C unwind"` ABI barrier is **still
-undefined behavior**. However, it is our **intent** to define that
-behavior in the future (and this is what the other roadmap items
-are all about). Additionally, we consider unwinding across an `extern "C"`
-boundary to be [LLVM-undefined behavior][LLVM-UB], whereas for
-`extern "C unwind"` it is not; i.e., `rustc` is not permitted to generate
-code that would _intentionally_ be undefined at the LLVM level in this case.
-
-In practical terms, the effect of using `"C unwind"` right now is that
-we will tell LLVM that the function "may unwind". We will also not add
-intentional shims that abort the program if unwinding occurs. (With a
-`"C"` ABI, we sometimes do both of those things.)
-
-Further, in practical terms, you can use the `"C unwind"` ABI today to
-enable a Rust panic to propagate across native frames if you like --
-but your program is relying on unspecified and undefined behavior
-which **likely will change** across Rust stable releases. Therefore,
-you will have to keep up. Effectively, you're on a nightly release,
-even though you're using only stable syntax. (The same is true for
-many aspects of unsafe code.)
-
-[LLVM-UB]: ../spec-terminology.md#LLVM-undefined-behavior-or-LLVM-UB
+roadmap item alone **only** adds the ABI string to Rust. All details
+of how that ABI strings are implemented for various targets are still
+considered [To Be Defined]. As a result, programs that rely on those
+details are not truly stable; you may find that the details change as
+Rust evolves. Eventually, though, we do intend to define many (but not
+all) aspects of how Rust panics and native unwinding interoperate.
+Moreover, we guarantee that unwinding will **not** result in
+[Undefined Behavior] and in particular not [LLVM-UB].
```

Comment:
This is more of an idea for the future, but maybe we should use a term like "optimizer-UB" or "intermediate-representation-UB" instead of LLVM-UB. The core idea of the term, I think, is that we are avoiding the form of UB that lets compilers make "adversarial" optimizations (in particular, in our case, eliminating landing pads).